app/uk/gov/hmrc/minorentityidentificationfrontend/services/StorageService.scala

```diff
@@ -125,16 +125,22 @@ class StorageService @Inject()(connector: StorageConnector) {
       val identifiersMatchBlock: JsObject =
         Json.obj("identifiersMatch" -> optIdentifiersMatch.contains(SuccessfulMatch))
 
-      Json.obj(
-        "businessVerification" -> Json.toJson(BusinessVerificationNotEnoughInformationToChallenge)(writeForJourneyContinuation),
-        "registration" -> Json.toJson(RegistrationNotCalled)(regFormat.writes)
-      ) ++
+      val businessVerificationStatusBlock: JsObject = {
+        val value = optBVStatus match {
+          case None           => BusinessVerificationStatus.writeForJourneyContinuation(BusinessVerificationNotEnoughInformationToChallenge)
```

Comment:
If the businessVerificationCheck is false here we need it to be an empty json object